core/src/test/java/org/apache/iceberg/MetadataTableScanTestBase.java

```diff
@@ -79,24 +80,19 @@ protected void validateTaskScanResiduals(TableScan scan, boolean ignoreResiduals
     }
   }
 
-  protected void validateIncludesPartitionScan(
-      CloseableIterable<FileScanTask> tasks, int partValue) {
-    validateIncludesPartitionScan(tasks, 0, partValue);
-  }
-
-  protected void validateIncludesPartitionScan(
-      CloseableIterable<FileScanTask> tasks, int position, int partValue) {
+  protected void validatePartition(
```

Comment:
Can we keep the other method that delegated to this one with 0 as default ?  Having 0 as default made the calls a bit easier to read.  Just a minor style preference

Reply:
thank you for the suggestion. I added a new method like what we did in the past but renamed to `validateSingletonPartition` as it seem to be suitable to verify data files with partition size = 1. I hope you dont hate it :)